src/Quasar/QuasarF.purs

```diff
@@ -66,7 +66,8 @@ data QuasarF a
   | CompileQuery DirPath SqlQuery Vars (CompileResultR :~> a)
   | FileMetadata FilePath (Unit :~> a)
   | DirMetadata DirPath (Maybe Pagination) ((Array QResource) :~> a)
-  | ReadFile PrecisionMode FilePath (Maybe Pagination) (ExpiredContent JArray :~> a)
+  | ReadFile PrecisionMode FilePath (Maybe Pagination) (JArray :~> a)
+  | ReadFileCache PrecisionMode FilePath (Maybe Pagination) (ExpiredContent JArray :~> a)
```

Comment:
I guess `ReadFile` can be expressed using `ReadFileCache`, bu just mapping with `_.content` over it's result. Is there any reasons for introducing separate constructor vs updating ReadFile with `ExpiredContent`?

Reply:
How would that look code wise? I don't quite understand the explination

Reply:
https://github.com/Cmdv/purescript-quasar/pull/1

Reply:
sweeeet thanks I would never got parts of that!! 😄 
going to read through it see exactly what you did 👍